.github/scripts/run-targets.sh

```diff
@@ -25,11 +25,11 @@ do
 
   if [[ "$mpi_flag" == "--with-mpi" && ! " ${non_mpi_targets[*]} " =~ " $target " ]]
   then
-    # Two ranks with one thread each, default NPROMA
-    mpirun -np 2 bin/$target 1 100
```

Comment:
One question: Why is this necessary?

Reply:
Answered offline: the CLAW variants are hard-coded to a multiple to 64, so to get full coverage this is necessary. GTG.